### DISCUSSION TITLE
Add extra action button on a collection

### DISCUSSION THREAD

**Discussion by b4d4** [directus/directus#12015] [Archive]
How i can add a button near others? . My idea is to select some users of my collection and then send him a email. 

![image](https://user-images.githubusercontent.com/58529478/157077640-d5a73a51-408d-4fd8-a769-772a8db35f1b.png)

**Comment by rijkvanzanten** [comment]
There's currently no way to inject custom logic in that position. However, we are working on a feature that could make that possible very soon 🙂

**Reply by Jeyakumarrathnaselvi** [reply]
Any update here?

**Reply by Csaba19999** [reply]
Any update on that?

**Reply by david-zacharias** [reply]
@Csaba19999 as @licitdev stated below, you can use a flow with [manual trigger](https://docs.directus.io/app/flows/triggers.html#manual) to perform actions (among others sending mails) on a set of selected items. 

However, the action will be located at the indicated menu next to the icons for CUD-Operations, but within a menu (flow menu) which has a default icon (flash) you can not change.

**Comment by licitdev** [comment]
Directus [Flows](https://docs.directus.io/configuration/flows/#flows) had been released in `v9.12.0`.
You should be able to perform your required operation by adding a [manual trigger](https://docs.directus.io/configuration/flows/triggers/#manual).

**Comment by havardox** [comment]
The limitation of no-code platforms becomes apparent in situations like these. Adding a custom action button to a toolbar seems like a pretty common requirement for someone who wants to perform their own batch actions on items. But alas, you're limited by the strictly defined capabilities of the platform that you've chosen.

As the old saying goes, you can't have your cake and eat it too.
